Library/Homebrew/unpack_strategy/tar.rb

```diff
@@ -38,9 +38,10 @@ def extract_to_dir(unpack_dir, basename:, verbose:)
         end
 
         system_command! "tar",
-                        args:    ["xf", tar_path, "-C", unpack_dir],
+                        args:    ["xf", tar_path, "-C", unpack_dir, *extra_args],
                         verbose: verbose
       end
     end
+    require "extend/os/linux/gnu_tar" if OS.linux?
```

Comment:
Please put this at the very end of the file and make it `require "extend/unpack_strategy/tar"`. You will need to add that file and make it contain `require "extend/os/linux/unpack_strategy/tar" if OS.linux?`.

Reply:
Yep